Project3/index.html

```diff
@@ -0,0 +1,118 @@
+<link rel="stylesheet" href="styles.css">
+<link href="https://fonts.googleapis.com/css?family=BenchNine:300,400,700|Teko:300,400,500,600,700&display=swap"
+    rel="stylesheet">
+<title>Document</title>
+</head>
+
+<body>
+
+    <form action="">
+        <h2>IDENTITY</h2>
+        <label for="real-name">Real Name</label>
+        <input id="real-name" type="text" required>
+
+        <label for="alias"> Alias </label>
+        <input id="alias" type="text" required>
+
+        <label for="email"> Email </label>
+        <input id="email" type="text" required>
+
+        <h2>PERSONAL<br> CHARACTERISTICS</h2>
+
+        <label for="age"> Age </label>
+        <input id="age" type="number" required>
+
+        <fieldset class="left-column">
+            <legend>Category</legend>
+            <p>Please select as many as apply to your origin and how you gained your powers.</p>
+
+            <div>
+                <input name="category" id="alien" type="checkbox">
+                <label for="alien">Alien</label>
+            </div>
+
+            <div><input name="category" id="cyborg" type="checkbox">
+                <label for="cyborg">Cyborg</label></div>
+
+            <div><input name="category" id="experiment" type="checkbox">
+                <label for="experiment">Experiment</label></div>
+
+            <div> <input name="category" id="human" type="checkbox">
+                <label for="human">Human</label></div>
+
+            <div><input name="category" id="inhuman" type="checkbox">
+                <label for="inhuman">Inhuman</label></div>
+
+            <div><input name="category" id="mutant" type="checkbox">
+                <label for="mutant">Mutant</label>
+            </div>
+        </fieldset>
+
+        <fieldset class="right-column">
+            <legend>Gender</legend>
+            <div><input name="gender" id="woman" type="radio">
+                <label for="woman">Woman</label></div>
+
+            <div><input name="gender" id="man" type="radio">
+                <label for="man">Man</label></div>
+
+            <div><input name="gender" id="nonbinary" type="radio">
+                <label for="nonbinary">Nonbinary</label></div>
+
+            <div><input name="gender" id="genderqueer" type="radio">
+                <label for="genderqueer">Genderqueer/Genderfluid</label></div>
+
+            <div><input name="gender" id="no-answer" type="radio">
+                <label for="no-answer">Prefer not to answer</label></div>
+
+            <div><input name="gender" id="self-describe" type="radio">
+                <label for="self-describe">Prefer to self describe</label>
+                <input name="gender" id="self-describe-answer" type="text"></div>
+        </fieldset>
+
+        <section>
+            <h2>Powers & Abilities</h2>
+            <label for="powers">Describe Your Powers</label>
+            <textarea id="powers"></textarea>
+
+            <label for="abilities">Rate Your Abilities</label>
```

Comment:
This should be an element _other_ than `label`, since there is no related `input` or other form element. It could be a heading, a `p` element or, if all the `select`s that follow were wrapped in a `fieldset`, this could be the `legend`.